XamlControlsGallery/App.xaml

```diff
@@ -393,7 +393,6 @@
             <Style x:Key="AutoSuggestBoxStyle" TargetType="AutoSuggestBox">
                 <Setter Property="VerticalAlignment" Value="Top" />
                 <Setter Property="IsTabStop" Value="False" />
-                <Setter Property="TextBoxStyle" Value="{StaticResource CustomAutoSuggestBoxTextBoxStyle}" />
```

Comment:
CustomAutoSuggestBoxTextBoxStyle is now not being used by anything so I think it's safe to delete out of App.xaml.

Reply:
Removed that too in the latest commit. Now all "dead" styling code should be removed ^^